client/src/components/Player.tsx

```diff
@@ -128,6 +130,7 @@ const Player: React.FC<PlayerProps> = ({ currentlyPlayingSong, token }) => {
               onChange={onUserSlide}
               className={classes.slider}
```

Comment:
I can't comment on the style class itself but could you add `marginLeft: 10, marginRight: 10`  to `slider` on line 201 below for some padding between the line and times. Make sure it's outside of the `[theme.breakpoints.down("xs")]:` as we want it applied everywhere.